content/recipes/prisma.md

```diff
@@ -533,7 +533,7 @@ import { PrismaService } from './services/prisma/prisma.service';
 async function bootstrap() {
   ...
   const prismaService: PrismaService = app.get(PrismaService);
```

Comment:
```suggestion
  const prismaService = app.get<PrismaService>(PrismaService);
```

let's improve(?) this one as well.

Reply:
we can even change it to:

```suggestion
  const prismaService = app.get(PrismaService);
```

<> isn't required